src/components/Controls.vue

```diff
@@ -509,8 +509,14 @@ export default {
 		width: 44px;
 		height: 44px;
 
-		&:hover, &:focus {
-			background-color: rgba(127,127,127,0.25) !important;
+		&[data-popper-shown] {
+			background-color: rgb(245, 245, 245);
+			[data-themes=dark] & {
+				background-color: #212121;
+			}
+			&.button-vue--vue-primary {
+				background-color: var(--color-primary-element);
```

Comment:
This should already be set through the type itself

Reply:
I also need this style to override the background color of the filter button when the filter is checked

<img width="287" alt="Screen Shot 2023-11-20 at 5 34 31 PM" src="https://github.com/nextcloud/deck/assets/91102425/e523cf27-b438-457c-9eeb-d81fc543186b">